cli/setup_wif.py

```diff
@@ -205,14 +221,15 @@ def main():
     parser.add_argument("--pool_name", help="Custom workload identity pool name", default="splunk-identity-pool")
     parser.add_argument("--provider_name", help="Custom workload identity provider name", default="splunk-provider")
     parser.add_argument("--dry_run", help="Just print what commands will be executed", action="store_true")
+    parser.add_argument("--no_interactive", action="store_true")
 
     project_ids = []
-    args = parser.parse_args()
+    args = parser.parse_args(args)
```

Comment:
nit: do we really need to pass `args` to the `parse_args` here?
```suggestion
    args = parser.parse_args()
```

Reply:
yes, by default it take it from systeam args, and it is null, but from unit test we send it like that